front/src/styles/scss/applications/rollingStockEditor/_rollingStockForm.scss

```diff
@@ -281,20 +280,28 @@
     padding: 0.25rem;
     &.selected {
       &.selector-blue {
-        outline: 0.1rem solid var(--blue);
+        outline: 0.13rem solid var(--blue);
       }
       &.selector-pink {
-        outline: 0.1rem solid #fd2f7e; // TODO: Apply the color to the curves + different opacity on hover
+        outline: 0.13rem solid #fd2f7e; // TODO: Apply the color to the curves + different opacity on hover
```

Comment:
It would be nice to create a new var for this color, can you ask Nicolas if he agrees ?